consul/fsm.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/hashicorp/raft"
 )
 
+var msgpackHandle = &codec.MsgpackHandle{}
```

Comment:
Might be good to put a comment on here since it's just hangin' out. I see why you moved it to this file.